### ISSUE TITLE
Text fields (Search, Password) not working with Device lock enabled on Windows, Auth

### ISSUE THREAD

**Issue by TheNightRider12** [ente-io/ente#2262]
### Description

My issue is similar to this: https://github.com/ente-io/ente/issues/2053, However, the fix that was said is not working for me on my laptop despite doing the same steps. 

When I open Ente Auth and go to search for an entry, I am not able to type anything into the search bar. 
Like the issue I posted said, one of the solutions was to add an exclusion for Ente auth into my AV. I have done that on my laptop, but I am still not able to type anything into the search bar. 

### Version

3.0.17

### What product are you using?

Ente Auth

### What platform are you using?

Desktop - Windows

**Comment by TrulyFlawed** [comment]
This may only apply to me, but having my lockscreen setting turned off will entirely stop this issue from occurring at all. However, when turning on the lockscreen setting, I am only able to type stuff in the search bar after minimizing and reopening the window.

I am on desktop (Windows), and using v3.0.17

**Comment by Hweord** [comment]
I have same issue. It works when I do alt+tab. Solve this please.

**Comment by ilyak83** [comment]
Same issue.
Desktop - Windows,  v3.0.17.

**Comment by AzzieDev** [comment]
Following this thread as I also use Windows Hello with Ente Auth

**Comment by rareseu** [comment]
Same issue on version 3.0.18 with lockscreen setting turned on. For now, workarounds mentioned here works. Thank you, guys!

**Comment by levik-opsilient** [comment]
Same issue
Windows 10 desktop, can only type once I move the focus.

**Comment by sciamano89** [comment]
iOS with version 3.0.12 - still happening randomly (tapping searchbar doesn't do anything). Force closing the app usually fixes the issue though.

**Comment by ifedan-ed** [comment]
disabling lockscreen or minimizing and then reopening the window if enabled works.

**Comment by lunyx** [comment]
Noticed the same thing, I thought it was some issue with my new keyboard, but I was able to reliably reproduce by closing the app and opening it again.  It doesn't accept any key input until tabbing out and back in. 

On this topic, it would also be helpful if we didn't have to click the search button, then click into the field, before being able to type.  It shouldn't take two clicks for this.  If anything, since there's no other use for typing on the main screen, maybe even automatically enter search whenever the user starts typing?

**Comment by TrulyFlawed** [comment]
> On the topic of the search, it would also be helpful if we didn't have to click the search button, then click into the field, before being able to type.  It shouldn't take two clicks for this.  If anything, since there's no other use for typing on the main screen, maybe even automatically enter search whenever the user starts typing?

I concur with this but I would advise you make a separate issue for this as it's a feature request.

**Comment by lunyx** [comment]
> I concur with this but I would advise you make a separate issue for this as it's a feature request.

Good point.  I mentioned it here.  I don't think it's linking since it's a discussion.  https://github.com/ente-io/ente/discussions/1254

**Comment by TrulyFlawed** [comment]
Just figured I would add: this issue still persists in v4.0.0 on the Windows app. I notice that this only happens when using the native OS app lock functionality, but otherwise searching seems to work just fine when using Ente's custom PIN or passwords for locking/unlocking the app.

**Comment by kenmanheimer** [comment]
Likewise:

- Coming out of screen lock, typing in the search field has no effect (characters aren't echoed and list scope doesn't narrow)
- Alt-tab focus away and back or minimizing then restoring app window rectifies
- Still happening on v4.0.2 as well as v3.1.3

I would consider this a show-stopper problem. Even though the workarounds are easy, they're not obvious.

**Comment by digitalgenesiskompound** [comment]
Still having this issue in v4.1.0 with Windows Hello. 
- Alt + Tab the window fixes 
- Using built in PIN function fixes

Would really like to just use Windows Hello and be able to type immediately. Will be following this issue.

**Comment by ua741** [comment]
Related https://github.com/flutter/flutter/issues/151457

**Comment by cbautomotive** [comment]
Same issue

**Comment by dyt08** [comment]
I'm currently using version 4.3.2 and this issue is still occurring. temporary solution is to minimize and reopen the app

**Comment by konekoangel** [comment]
Issue is still present in v4.3.6

**Comment by miguedevel** [comment]
Finally fixed for me on 4.4.10

**Comment by TheNightRider12** [comment]
Fixed for me as well. Closing the issue.
